docs/add-your-app.md

```diff
@@ -0,0 +1,138 @@
+# Adding Your Apps
+
+
+## Automatic discovery
```

Comment:
Its weird you start the section describing a feature that is not supported. Why not simply begin with two ways of app discovery:
- manual discovery and steps underneath it.
- automatic discovery - not supported

Reply:
I would highly recommend listing the high level steps for quicker consumption. This sort of becomes a high level algorithm if people decide to automate this. Markdown TOC may help.

Reply:
Should you start with configuring the app registry?